drivers/hmis/app/graphql/mutations/create_service_category.rb

```diff
@@ -0,0 +1,32 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class CreateServiceCategory < BaseMutation
+    argument :name, String, required: true
+
+    field :service_category, Types::HmisSchema::ServiceCategory, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false, resolver: Resolvers::ValidationErrors
+
+    def resolve(name:)
+      raise 'access denied' unless current_user.can_configure_data_collection?
```

Comment:
We've been switching to this gradually, so that we can standardize access errors eventually 
```suggestion
      access_denied! unless current_user.can_configure_data_collection?
```